driver/apdu/qmi_qrtr.c

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: MIT
+/*
+ * Copyright (c) 2024, Luca Weiss <luca.weiss@fairphone.com>
+ */
+#include "qmi_qrtr.h"
+
+#include <euicc/interface.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <libqrtr-glib.h>
+#include "qmi_qrtr_helpers.h"
+
+#define UIM_SLOT 2 // TODO: Make this runtime-configurable
```

Comment:
Do you have a suggestion how to handle the dual-SIM situations, where one wants to set the slot to 1 or 2 (or in theory slot 3 and higher could also exist)?

One idea is adding an environment variable UIM_SLOT=1 to lpac invocation but this wouldn't be nice in my opinion.

Reply:
@estkme Thanks for merging!

But do you have any suggestion how to handle this part? I'm actually thinking now environment variable might be okay since also the AT backend just does `getenv("AT_DEVICE")` for example. 

I'd add a `getenv("UIM_SLOT")` call to this backend then to replace this hardcoded slot 2.

Reply:
I think `getenv("UIM_SLOT")`  is okay for now, later our team may have a guide to how to name an ENV var ---- that will be another pr.

Reply:
Done: https://github.com/estkme-group/lpac/pull/97